heareval/tasks/pipeline.py

```diff
@@ -88,37 +88,48 @@ def requires(self):
     def run(self):
         archive_path = self.requires()["download"].workdir.joinpath(self.infile)
         archive_path = archive_path.absolute()
-        output_path = self.workdir
-        if self.outdir is not None:
-            output_path = output_path.joinpath(self.outdir)
+        output_path = self.workdir.joinpath(self.outdir)
```

Comment:
The default for `self.outdir` is None -- if this parameter is non-optional (i.e. self.outdir should never be None), then we should remove the default above.

Reply:
Removing optionality